helpers/objects.js

```diff
@@ -1,5 +1,19 @@
-function singleSort(arrayOfObjs, sortProperty1) {
-	return null;
+function getMatching(arrayOfObjs, matchProperty, matchValue) {
+	return arrayOfObjs.filter((entry) => {
+		return entry[matchProperty] === matchValue ? entry : null;
+	});
+}
+
+function getFirstMatching(arrayOfObjs, matchProperty, matchValue) {
```

Comment:
What's the reason for calling getFirstMatching instead of calling getMatching and picking the first item?

Reply:
🤔

Reply:
Fair point, fixed
Not sure why I wrote both, I'm sure it made sense at the time :|